doc: update known issues (#433)
nrfconnect/sdk-sidewalk: doc/sidewalk_known_issues.rst

v1.14.5-dev1
------------

> * KRKNWK-17860: QSPI assert when trying to perform dfu in sidewalk application mode
>     The DFU should be performed only in dfu mode. However the dfu bluetooth service is available in sidewalk mode, and using it lead to assertion failure witch means a zephyr fatal error.

Reviewer: ```suggestion
* KRKNWK-17860: QSPI assert occurs when performing DFU in the Sidewalk application mode
    The DFU should be performed only in the DFU mode. 
    The DFU Bluetooth service can be used in the Sidewalk mode, however, using it leads to assertion failure, resulting in a zephyr fatal error.
```

> * KRKNWK-17800: After network connection is back end device cannot find route to host.
>     After losing connection to Sidewalk servers, sensor demo app over ble stuck with error -38 (SID_ERROR_NO_ROUTE_AVAILABLE). Device need to be reset manually.

Reviewer: ```suggestion
* KRKNWK-17800: After reconnecting to network, the end device cannot find the route to its host
    After the device disconnects from Sidewalk servers, the Sensor monitoring app over Bluetooth LE shows an error ``-38 (SID_ERROR_NO_ROUTE_AVAILABLE)``. 
    To fix the error, the device needs to be reset manually.
```

Please double-check the meaning here, not sure if I managed to keep it

> * KRKNWK-17750: Sidewalk errors on multiple messages send in a short time period.
>     When sending multiple sidewalk messages in short period of time internal queues may be full and cause misleading errors messages like -12 (SID_ERROR_INCOMPATIBLE_PARAMS). The message should be resend after some amount of time needed for protocol to empty the queues.

Reviewer: ```suggestion
* KRKNWK-17750: Error occurs when sending multiple messages in a short period of time
    When sending multiple Sidewalk messages in a short period of time, the internal queues might become full, showing misleading error messages, such as ``-12 (SID_ERROR_INCOMPATIBLE_PARAMS)``. 
    The message should resend after some time needed for protocol to empty the queues.
```

> * KRKNWK-17244:  CMake warnings during Sidewalk samples build.
>     "No SOURCES given to Zephyr library" CMake warnings present in build log of Sidewalk application.

Reviewer: ```suggestion
* KRKNWK-17244:  CMake warnings when building the Sidewalk samples
    ``No SOURCES given to Zephyr library`` CMake warnings shows in the build log of Sidewalk application.
```
Is there anything additional that can be added to this? E.g., what should user do when they see it? Will it create any other issues?

Author: Added an extra sentence

> * KRKNWK-17374: Sporadic Zephyr fatal error after disconnecting on FSK
>     After disconnecting on FSK, Zephyr fatal error occurs due to assertion in semaphore module.
>     The error reproduces rarely - once per few days.
>     In the release mode the device resets automatically, however, in the debug mode it need to be reset manually.
>     Currently, this issue occurs for Sidewalk 1.14 libraries and it will be fixed in the version 1.16.

Reviewer: ```suggestion
* KRKNWK-17374: Sporadic Zephyr fatal error after disconnecting on FSK
    After disconnecting on FSK, Zephyr fatal error occurs due to assertion in semaphore module.
    The error reproduces rarely - once per few days.
    The device resets automatically in the release mode, however, in the debug mode it needs to be reset manually.
    Currently, this issue occurs for Sidewalk 1.14 libraries and it will be fixed in the version 1.16.
```
I know this item was already here, but did some minor rewording

v1.14.5-dev1
------------

* KRKNWK-17860: QSPI assert occurs when performing DFU in the Sidewalk application mode
    The DFU should be performed only in the DFU mode.
    The DFU Bluetooth service can be used in the Sidewalk mode, however, using it leads to assertion failure, resulting in a Zephyr fatal error.

* KRKNWK-17800: After reconnecting to network, the end device cannot find the route to its host
    After the device disconnects from Sidewalk servers, the Sensor monitoring app over Bluetooth LE shows an error ``-38 (SID_ERROR_NO_ROUTE_AVAILABLE)``.
    To fix the error, the device needs to be reset manually.

* KRKNWK-17750: Error occurs when sending multiple messages in a short period of time
    When sending multiple Sidewalk messages in a short period of time, the internal queues might become full, showing misleading error messages, such as ``-12 (SID_ERROR_INCOMPATIBLE_PARAMS)``.
    The message should resend after some time needed for protocol to empty the queues.

* KRKNWK-17244:  CMake warnings when building the Sidewalk samples

>     ``No SOURCES given to Zephyr library`` CMake warnings shows in the build log of Sidewalk application. Application builds successfully, but this may obfuscate another important warnings.

Reviewer: ```suggestion
    CMake warnings (``No SOURCES given to Zephyr library``) show in the build log of Sidewalk application. 
    Application builds successfully, but the error may obfuscate other important warnings.
```
Reworded it a bit since it's the item is a bit longer now
